server/service/src/stock_line/update.rs

```diff
@@ -95,6 +99,36 @@ fn generate(
     existing
 }
 
+fn log_stock_changes(
+    ctx: &ServiceContext,
+    existing: StockLineRow,
+    new: StockLineRow,
+) -> Result<(), RepositoryError> {
+    if existing.location_id != new.location_id {
```

Comment:
could I add in another request? It would be awesome to have the values as well.. 
Would require adding a column to the `activity_log` table. In its simplest form, just a text column that has `changed from {} to {}`. The mSupply log has a column `event` of type text ( and a `data_JSON` of type text as well. Most info is in the `event` column tho ).

I know that @andreievg wanted an enum for the event, which is what we have in `activity_log` - and which is what you've done here. I think it is valid to have supplemental information in another column though, as well as the enum. We'd need to expose the column in the log query too.. so that we can show the data for the stock line

Reply:
I was thinking about having a text field for this as well! Would also reduce the enum to just StockChange and have the changes in text instead of the enum 🤔

Reply:
I suspect that @andreievg would prefer to keep the full enum.. but don't want to speak on his behalf! Either is fine by me 😁

Reply:
No problem ~ either is fine (: just thought the Enum list will get really long 😄